From f86eb2b3a48faca0eaffc957aa5e624957740e5d Mon Sep 17 00:00:00 2001
Message-ID: <f86eb2b3a48faca0eaffc957aa5e624957740e5d.1752951853.git.matahina@posteo.net>
From: matahina <matahina@posteo.net>
Date: Sat, 19 Jul 2025 21:03:45 +0200
Subject: [PATCH] the_commit!

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index af0ca67af..344133c5d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -505,7 +505,7 @@ endif()
 # Set some compiler options
 if(UNIX OR MINGW)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++17 -Ofast")
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wno-unused-function -funroll-loops -march=icelake-client")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wno-unused-function -funroll-loops")
 endif()
 
 if(MINGW AND CMAKE_BUILD_TYPE MATCHES Release)
-- 
2.50.1

